javascript/modules/client/src/client.ts

```diff
@@ -489,6 +474,24 @@ export class Client extends ClientCore implements IClient {
 
     return Promise.resolve({ deposits, withdrawals });
   }
+  /**
+   * Build withdraw action
+   *
+   * @private
+   * @param {string} to
+   * @param {bigint} value
+   * @param {IWithdrawParams} params
+   * @return {*}  {DaoAction}
+   * @memberof Client
+   */
+  private _buildActionWithdraw(params: IWithdrawParams): DaoAction {
+    // TODO: CONFIRM THAT THE "to" field needs to contain the DAO address
+    return {
+      to: AddressZero,
```

Comment:
I think we should put the DAO's address in this field

Reply:
Should we pass the dao address as a `param` then?

Reply:
I think so, unless the client already knows the DAO's address.

Reply:
:+1: